Tasks/ServiceFabricDeploy/ServiceFabricSDK/Utilities.ps1

```diff
@@ -21,12 +21,12 @@
         $Destination
     )
 
-    if (!(Test-Path $File))
+    if (!(Test-Path -LiteralPath $File))
```

Comment:
I think occurences of Get-Path also need change

Reply:
Get-Content as well

Reply:
updated with fix for get-content and copy-item. get-path is not used anywhere.